en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md

```diff
@@ -69,6 +69,19 @@ flow.statistics.enable=true
 stat.tracer.collect_payloads=true
 stat.tracer.collect_mediation_properties=true
 ```
+### Configuring the Micro Integrator to publish data to EI Analytics
```

Comment:
### Configuring the Micro Integrator to publish data to EI Analytics

Analytics publishing can be configured in the `[monitoring]` section of the `<MI_HOME>/conf/deployment.toml` file as shown below.

!!! Note
    By default, the Micro Integrator is internally configured (with the following) to connect with an EI Analytics server running on the same VM. To change the default setup, you need to add the following to the `deployment.toml` file and update the values.

```toml
[monitoring]
ei_analytics.server_url = "tcp://localhost:7612"
ei_analytics.auth_server_url = "ssl://localhost:7712"
ei_analytics.username = "admin"
ei_analytics.password = "admin"        
```

If you have a cluster of Analytics nodes (running on the same VM or on different VMs), the `server_url` and `auth_server_url` can be configured to balance the load among the different server instances. See [Enabling Load Balancing Data Agent for Publishing to Analytics](#enabling-load-balancing-data-agent-for-publishing-to-analytics) given below for details.